pkg/endpoint/endpoint.go

```diff
@@ -983,6 +983,14 @@ func (e *Endpoint) LogStatus(typ StatusType, code StatusCode, msg string) {
 	// log message to a single writer?
 	e.Status.indexMU.Lock()
 	defer e.Status.indexMU.Unlock()
+	e.logStatusLocked(typ, code, msg)
+}
+
+func (e *Endpoint) LogStatusOK(typ StatusType, msg string) {
```

Comment:
Put this method private.

Reply:
I can't :( It's called twice by endpointmanager (and only once by endpoint!).

I could replace all the calls with `LogStatus(typ, OK, msg)` instead. What do you think?

Reply:
Let's keep it as-is. We already noticed that the separation of endpoint and endpointmanager is creating a lot of complication. It seems like the right step to move them to a single package. They depend on each other. Once that's done, issues such as this will resolve itself.